### PR TITLE
feat: Added more granular timestamp for Desktop logs

### PR DESCRIPTION
**Pull request by bs-amitmahajan29**
Task: https://browserstack.atlassian.net/browse/DATA-7249

### CODE REVIEW

File: lib/chitragupta/util.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def sanitize_keys(log_level, timestamp, message, filename)
       data = initialize_data(message)
 
       data[:log][:level] = log_level
-      data[:meta][:timestamp] = timestamp.iso8601() rescue timestamp
+      data[:meta][:timestamp] = timestamp.iso8601(3) rescue timestamp
```
Comment (nimishbansal-bs) [inline lib/chitragupta/util.rb:13]: not bumping the version?

Comment (bs-amitmahajan29) [reply lib/chitragupta/util.rb:13]: Good catch, bumped.

**Review by nimishbansal-bs — APPROVED** [review]

**Comment by bs-amitmahajan29** [comment]
RUN_SCA

**Comment by bs-amitmahajan29** [comment]
RUN_TESTS

**Review by aashishsingh2803 — DISMISSED** [review]

File: lib/chitragupta/util.rb (hunk 1 above)
Comment (bs-amitmahajan29) [inline lib/chitragupta/util.rb:13]:
```suggestion
      data[:meta][:timestamp] = timestamp.iso8601(4) rescue timestamp
```

File: lib/chitragupta/util.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ def sanitize_keys(log_level, timestamp, message, filename)
       data = initialize_data(message)
 
       data[:log][:level] = log_level
-      data[:meta][:timestamp] = timestamp.iso8601() rescue timestamp
+      data[:meta][:timestamp] = timestamp.iso8601(4) rescue timestamp
```
Comment (bs-amitmahajan29) [inline lib/chitragupta/util.rb]:
```suggestion
      data[:meta][:timestamp] = timestamp.iso8601(3) rescue timestamp
```

**Pull request closed without merging**